libsolidity/grammar.txt

```diff
@@ -43,13 +43,13 @@ StorageLocation = 'memory' | 'storage'
 Block = '{' Statement* '}'
 Statement = IfStatement | WhileStatement | ForStatement | Block |
             ( PlaceholderStatement | Continue | Break | Return |
-              Throw | SimpleStatement | ExpressionStatement ) ';'
+              Throw | SimpleStatement ) ';'
 
 ExpressionStatement = Expression | VariableDefinition
```

Comment:
I rather think that this one here is wrong: An ExpressionStatement is a statement that only contains an expression. A VariableDefinition is not an expression.